docs/install-mac-cpu.md

```diff
@@ -0,0 +1,9 @@
+```console
+pip install -U pip
+pip install -U setuptools wheel
+
+# pytorch documentation - https://pytorch.org/get-started/locally/
+pip install torch==1.13.1 torchvision==0.14.1 -f https://download.pytorch.org/whl/cpu/torch_stable.html
+
+pip install autogluon
+```
```

Comment:
Need to also update the source install instructions

Reply:
done